src/components/toc.js

```diff
@@ -6,17 +6,14 @@ import style from "./toc.module.css"
 const HEADER_HEIGHT = 85
 const EXTRA_MARGIN = 50
 const INTERSECTION_THRESHOLD = 1
+const HEADERS = "article h2, article h3, article h4, article h5, article h6"
```

Comment:
I like the idea of a constant. 👍🏾 I think `HEADERS` more clearly expresses what is being observed, though.